File: VB/WindowsFormsApplication3/Form1.vb
Imports DevExpress.XtraReports.UI
Imports System
Imports System.Windows.Forms

Namespace WindowsFormsApplication3
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
            Dim pt As New ReportPrintTool(New XtraReport1())
            pt.ShowPreviewDialog()
        End Sub
    End Class
End Namespace

File: CS/WindowsFormsApplication3/Form1.cs
using DevExpress.XtraReports.UI;
using System;
using System.Windows.Forms;

namespace WindowsFormsApplication3 {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e) {
            ReportPrintTool pt = new ReportPrintTool(new XtraReport1());
            pt.ShowPreviewDialog();
        }
    }
}
